code/style.css

```diff
@@ -1,58 +1,214 @@
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background:black;
+  color: white;
+  background: url("castle.jpg");
+  background-size: 100% 100%;
 }
 
-p {
-  margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+header {
+  background: white;
+  opacity: 50%;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+  border-bottom: 1px solid;
+}
+
+header a {
+  color: black;
+  font-size: 20px;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+header h1 {
+  color: black;
+  font-size: 50px;
+}
+
+header img {
+  height: 100px;
+  margin:  0 24px 0 10px;
+}
+
+.cards p {
+  font-size: 20px;
+  margin-left: 10px;
+  font-weight: bold;
+  border-radius: 5px;
+  margin-top:0px;
+  width: 150px;
 }
 
 .cards {
-  margin: 50px auto;
+  margin: 30px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  height: auto; 
+  background: transparent;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  flex-wrap: wrap;
+  background-image: url(assets/cluedo_cover.jpeg);
```

Comment:
Let's wrap it up with quote marks, like that :
```
background-image: url("assets/cluedo_cover.jpeg");
```
Some browsers might not approve this value without quote marks